package.json

```diff
@@ -40,6 +40,7 @@
     "style-loader": "^1.1.3",
     "stylelint-config-wikimedia": "^0.10.1",
     "stylelint-webpack-plugin": "^2.1.0",
+    "url-loader": "^4.1.0",
```

Comment:
Could you point out where this is being used?

Reply:
It's for the body of the preview loading state, which is currently an image (.png) but it's meant be animated (.gif)

https://github.com/wikimedia/wikipedia-preview/blob/T256309-loading-state/style/preview.less#L98

However, I synced briefly with Sudhanshu today and we think it might be better to achieve the animation with CSS rather than a gif, kind of like we are doing with [the spinner](https://github.com/wikimedia/wikipedia-preview/blob/T256309-loading-state/style/popup.less#L180). In that case we wouldn't need `url-loader`. I'm gonna look in the CSS animations now, will keep you updated

Reply:
Added animation, removed `url-loader`